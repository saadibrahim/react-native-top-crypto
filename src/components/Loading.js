import React, { Component } from 'react';
import { View, ActivityIndicator } from 'react-native';
import styles from './styles/LoadingStyles';

export default class Loading extends Component {
  render() {
    return (
      <View style={styles.container}>
        <ActivityIndicator animating size="large" color="#2CBCF2" />
      </View>
    );
  }
}
