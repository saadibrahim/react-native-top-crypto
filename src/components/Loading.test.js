import React from 'react';
import renderer from 'react-test-renderer';

import Loading from './Loading';

test('renders the Navbar using Snapshots', () => {
  const component = renderer.create(<Loading />);
  expect(component).toMatchSnapshot();
});
