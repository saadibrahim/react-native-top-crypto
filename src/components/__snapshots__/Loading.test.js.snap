// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Navbar using Snapshots 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#2CBCF2"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
